mantidproject/mantid
# Rename tabs on Indirect Corrections

Fixes #13104.

To test:
- Open the Indirect Corrections interface
- See that the tabs have more descriptive names now
- See that the documentation is updated
